ci/release/update-version.sh

```diff
@@ -62,6 +54,16 @@ sed_runner "s/VERSION_NUMBER=\".*/VERSION_NUMBER=\"${NEXT_SHORT_TAG}\"/g" ci/bui
 # Need to distutils-normalize the original version
 NEXT_SHORT_TAG_PEP440=$(python -c "from setuptools.extern import packaging; print(packaging.version.Version('${NEXT_SHORT_TAG}'))")
 
+# bump rapids libraries
+for FILE in dependencies.yaml conda/environments/*.yaml; do
+  sed_runner "/- &cudf_conda cudf==/ s/==.*/==${NEXT_SHORT_TAG_PEP440}\.*/g" ${FILE}
+  sed_runner "/- cudf==/ s/==.*/==${NEXT_SHORT_TAG_PEP440}\.*/g" ${FILE}
```

Comment:
for now there is a bit of duplication necessary here for `cudf`.

it will be removed in the future when we revamp these scripts and the process around them.